UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/AzureADB2CAuthenticationTests.cs

```diff
@@ -11,20 +11,27 @@ namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests
     class AzureADB2CAuthenticationTests
     {
         private ExchangeSetApiClient ExchangeSetApiClient { get; set; }
+        private FssApiClient FssApiClient { get; set; }
         private TestConfiguration Config { get; set; }
-        private string EssB2CToken { get; set; }        
+        private string EssB2CToken { get; set; }
         private string EssB2CCustomizedToken { get; set; }
         public DataHelper DataHelper { get; set; }
-
+        private string FssJwtToken { get; set; }
+        private readonly List<string> cleanUpBatchIdList = new List<string>();
```

Comment:
Can casing of "cleanUpBatchIdList" changed to pascal case "CleanUpBatchIdList" in all test case files to maintain consistency.

Also change casing of sinceDateTime to SinceDateTime in current file.

Remove line breaks at 23/24

Reply:
Did changes accordingly. Thanks!